src/features/rewards/walletWrapper/index.tsx

```diff
@@ -83,7 +83,17 @@ export interface ActionWallet {
   testId?: string
 }
 
-export type NotificationType = 'ads' | 'ads-launch' | 'backupWallet' | 'contribute' | 'grant' | 'insufficientFunds' | 'tipsProcessed' | 'error' | ''
+export type NotificationType =
+  'ads' |
+  'ads-launch' |
+  'backupWallet' |
+  'contribute' |
+  'grant' |
+  'insufficientFunds' |
+  'tipsProcessed' |
+  'error' |
+  'pendingContribution' |
+  ''
```

Comment:
I know this is just alignment mod but do we need the `''`. The `default` case already handles

Reply:
isn't default case right for this use case, when you have empty string as type can't be null